tokio/tests/no_rt.rs

```diff
@@ -7,8 +7,8 @@ use futures::executor::block_on;
 use std::net::TcpListener;
 
 #[test]
-#[should_panic(expected = "no timer running")]
-fn panics_when_no_timer() {
+#[should_panic(expected = "no reactor running")]
```

Comment:
It seems like this should fail with the same standardized error message as the other errors.
```suggestion
#[should_panic(expected = "there is no reactor running, must be called from the context of a Tokio 1.x runtime")]
```